src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -629,6 +630,10 @@ await using (var actions = _destination.DatabaseRecord())
                 {
                     await actions.WriteDatabaseRecordAsync(databaseRecord, result, _options.AuthorizationStatus, _options.OperateOnDatabaseRecordTypes);
                 }
+                catch (LicenseLimitException lle)
+                {
+                    throw new LicenseLimitException(lle.Type, lle.Message);
```

Comment:
What is the point of rethrowing the same exception? Please note that you're loosing the original stacktrace then

Reply:
We want it to stop the import and not be cached afterward that will just add an error and continue with the import

Reply:
I don't follow. Wouldn't the original exception stop the import anyway? @grisha-kotler ?

Reply:
Oh, I see now what you want to achieve. But then just do:

```
catch (LicenseLimitException)
{
    throw;
}
```

This way you keep the original stackrace

Reply:
Done